### DISCUSSION TITLE
JetBrain's DataGrip - the Swiss Army knife for database management/maintenance.

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#28] [Show and tell]
My go-to tool/IDE for database management/maintenance is JetBrain's DataGrip. For me, it's the Swiss Army knife for database management/maintenance. Regardless of the vendor, this IDE supports many [No]SQL databases... look at this amazing list!
Part 1:
![image](https://github.com/holgerflick/flixaccounting/assets/681882/4bcc522d-9537-4c1f-b4f7-4f3ab541e398)
Part 2:
![image](https://github.com/holgerflick/flixaccounting/assets/681882/6018855d-b8af-4b0d-8604-031411e39c07)

What is your opinion on this? Do you use other programs for these tasks?

**Comment by holgerflick** [comment]
See the next How it works with Holger ;-) The part that shows the database model...

**Reply by hindermath** [reply]
In any case, I won't miss your new HIWWH video series! :-D
